juriscraper/opinions/united_states/administrative_agency/bia.py

```diff
@@ -0,0 +1,107 @@
+"""Scraper for Dept of Justice. Board of Immigration Appeals
+CourtID: bia
+Court Short Name: Dept of Justice. Board of Immigration Appeals
+Author: William Palin
+Reviewer:
+Type:
+History:
+    2021-12-18: Created by William E. Palin
+"""
+import re
+from typing import Any, Dict
+
+from juriscraper.OpinionSiteLinear import OpinionSiteLinear
+
+
+class Site(OpinionSiteLinear):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.court_id = self.__module__
+        self.url = "https://www.justice.gov/eoir/ag-bia-decisions"
+        self.article = None
+        self.volume = 0
+        self.urls = None
+
+    def _process_elements(self, elements) -> Dict[str, Any]:
+        """Process the element grouping.
+
+        There is no easy way to parse our the content. Unfortunately, the DOJ
+        admins randomly nest elements and not others. The only consistency
+        is that they content is always split between HR tags.  So we iterate
+        over the elements in order until we find an HR tag and then process
+        the content.  Rinse Wash Repeat.
+
+        Additionally, the only date we have is the year of the decision.
+
+        :param elements: The elements between <hr> tags.
+        :return: Case data
+        """
+        case = {}
+        bold_text = elements[0].xpath(".//strong[1]/.. | .//b[1]/..")
+        if not bold_text:
+            return {}
+        intro_text = (
+            elements[0].xpath(".//strong[1]/.. | .//b[1]/..")[0].text_content()
+        )
+        intro_text = intro_text.replace(";", ",")
+        name, cite = intro_text.split(",", 1)
+        # Unfortunately there are no accessible file dates without PDF parsing
+        # So we generate a date and mark it as date_filed_is_approximate = True
+        case["date_filed_is_approximate"] = True
```

Comment:
What about using the thing we use at the tax court to get the date values from the parsed PDFs?

Reply:
Set this up with tax code

Reply:
Left this as it was because a date is needed (I think)